docs/source/model_design/concept_model/monte_carlo_uncertainty/index.rst

```diff
@@ -29,8 +29,97 @@ Monte Carlo Uncertainty in Vivarium
 
 .. contents::
    :local:
-   :depth: 1
+   :depth: 2
 
-.. todo:: 
+What is Monte Carlo Uncertainty
+--------------------------------
 
-  This section. Link to James' presentation?
\ No newline at end of file
+.. todo::
+
+  Link to some good existing resources. Anybody know of any good ones?
+
+Uncertainty Parameterization in Vivarium
+-----------------------------------------
+
+What is a draw?
+++++++++++++++++
+
+We can think of a draw as a single iteration of some calculation or model run within a series of calculations or model runs that make up the summary estimates. 
```

Comment:
I'm not sure I understand what is meant by "iteration" here. I don't know the clearest way to explain what a draw is, but I'd start with (or at least include) that a draw is a sample from a probability distribution and that we use the 1,000 draws/samples to represent the distribution, instead of using a continuous density function.

Reply:
Yes, this is much more clear. I updated to include this language and also added in a link to a monte carlo overview resource which will hopefully also provide some more context. Thanks!